wolfcrypt/src/ecc.c

```diff
@@ -9426,8 +9432,8 @@ int ecc_mul2add(ecc_point* A, mp_int* kA,
       }
 
       if (err == MP_OKAY) {
-        /* if it's 2 build the LUT, if it's higher just use the LUT */
-        if (idx1 >= 0 && fp_cache[idx1].lru_count == 2) {
+        /* if it's >=2 build the LUT, if it's higher just use the LUT */
```

Comment:
Shouldn't this comment now read

"if it's >= 2 AND the LUT is not set build the LUT,"

Reply:
Changed comment to reflect current flow.